macros/blueprint-proc-macro/src/job.rs

```diff
@@ -278,7 +255,7 @@ pub(crate) fn generate_event_workflow_tokenstream(
                     // If is_raw, assume the actual context is the second param
                     quote! { ctx. #field_in_self .clone() }
                 })
-                .expect("No context found");
+                .ok_or_else(|| syn::Error::new(Span::call_site(), "Must specify a context"))?;
 
```

Comment:
you should use `params` here for the instead of the `Span::call_site()` so the errors appears on the function args.